### PR TITLE
vcf-report v2.4.1 easyconfig

### PR DESCRIPTION
**Pull request by dennishendriksen**

### CODE REVIEW

File: easybuild/easyconfigs/v/vcf-report/vcf-report-v2.4.1-Java-11-LTS.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name = 'vcf-report'
+version = 'v2.4.1'
+versionsuffix = '-Java-%(javaver)s-LTS'
+
+homepage = 'https://github.com/molgenis/vip-report/'
+description = """Command-line application to generate a report for any VCF (Variant Call Format) file based on a report template"""
+
+toolchain = {'name': 'system', 'version': ''}
```
Comment (pneerincx) [inline easybuild/easyconfigs/v/vcf-report/vcf-report-v2.4.1-Java-11-LTS.eb:8]: https://docs.easybuild.io/en/latest/System_toolchain.html#system-constant

**Review by pneerincx — CHANGES_REQUESTED** [review]
Why not use the "highly recommended" constant?

**Comment by dennishendriksen** [comment]
I followed the instructions in the `Using the system compiler` section of https://docs.easybuild.io/en/latest/System_toolchain.html#using-the-system-compiler which states:
```
To use the system toolchain in an easyconfig file, simply use 'system' as toolchain name:

toolchain = {'name': 'system', 'version': ''}
```

**Comment by pneerincx** [comment]
Please also read the `SYSTEM constant` section directly below `Using the system compiler`.

**Comment by dennishendriksen** [comment]
I did read it after your first comment. I've picked the initial solution that was described in the documentation. Since the modules have been deployed to multiple environments already I don't think it is worth it to rebuild and go through the process again.

**Review by pneerincx — APPROVED** [review]

**Comment by dennishendriksen** [comment]
> pneerincx approved these changes 10 days ago
ok, can you merge it as well? ;-)

**Pull request merged**